Makefile

```diff
@@ -63,6 +63,7 @@ test-coverage:
 	docker compose -f dev/docker-compose-integration.yml up -d
 	sh ./dev/run-azurite.sh
 	sh ./dev/run-gcs-server.sh
+	sleep 10
```

Comment:
need this similar to in `test-integration`. when docker contains are already created, `provision.py` will fail because underlying docker services are not yet spun up

Reply:
Filed a feature improvement request in #637, to get rid of this in the future